# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/cmake/hotfix-uname.patch
# Copyright (C) 2006 - 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

Cmake combines stderr and stdout, as T2 emmits a warning to stderr
we need to get rid of it.

  - Rene Rebe <rene@exactcode.de>

--- cmake-3.26.2/Modules/CMakeDetermineSystem.cmake.vanilla	2023-03-31 12:51:31.432782357 +0200
+++ cmake-3.26.2/Modules/CMakeDetermineSystem.cmake	2023-03-31 12:56:28.525781053 +0200
@@ -48,13 +48,15 @@
       unset(_CMAKE_HOST_SYSTEM_MAJOR_VERSION)
       unset(_CMAKE_HOST_SYSTEM_MINOR_VERSION)
     else()
-      execute_process(COMMAND ${CMAKE_UNAME} -r
+      execute_process(COMMAND ${CMAKE_UNAME} -r
+        ERROR_VARIABLE DEV_NULL
         OUTPUT_VARIABLE CMAKE_HOST_SYSTEM_VERSION
         OUTPUT_STRIP_TRAILING_WHITESPACE
         ERROR_QUIET)
     endif()
     if(CMAKE_HOST_SYSTEM_NAME MATCHES "Linux|CYGWIN.*|MSYS.*|^GNU$|Android")
-      execute_process(COMMAND ${CMAKE_UNAME} -m
+      execute_process(COMMAND ${CMAKE_UNAME} -m
+        ERROR_VARIABLE DEV_NULL
         OUTPUT_VARIABLE CMAKE_HOST_SYSTEM_PROCESSOR
         RESULT_VARIABLE val
         OUTPUT_STRIP_TRAILING_WHITESPACE
@@ -104,7 +104,8 @@
         OUTPUT_STRIP_TRAILING_WHITESPACE
         ERROR_QUIET)
     else()
-      execute_process(COMMAND ${CMAKE_UNAME} -p
+      execute_process(COMMAND ${CMAKE_UNAME} -p
+        ERROR_VARIABLE DEV_NULL
         OUTPUT_VARIABLE CMAKE_HOST_SYSTEM_PROCESSOR
         RESULT_VARIABLE val
         OUTPUT_STRIP_TRAILING_WHITESPACE
